src/plat/qemu-arm-virt/config.cmake

```diff
@@ -34,9 +37,32 @@ endmacro()
 
 if(KernelPlatformQEMUArmVirt)
 
+    # Our default QEMU configuration is AARCH64/Cortex-a53. If KernelSel4Arch is
+    # set to aarch32, the default is Cortex-a15. Setting ARM_CPU allows picking
+    # a specific CPU and forces KernelSel4Arch to the respective architecture.
+    # Note that the QEMU virt platform has some limitations:
+    # - only specific ARMv7 and ARMv8 cores can be used.
+    # - ARMv8 cores can't be started in AARCH32 mode
+    # - Cortex-A32 that implements AARCH32 only is not suppoted
```

Comment:
I think I would remove this section, as it doesn't really help and can get out-of-date easily.

If you do keep it, I would say "qemu-arm-virt platform" instead of "QEMU virt platform" if the limitation is in seL4 and not QEMU, otherwise it's ambiguous.

Reply:
The QEMU platform is "virt". I've added a line with a link to the full QEMU documentation for this platform now. I prefer to keep this comment, as one can be mislead easily to assume the virt platform can emulate everything.

Reply:
If these are limitations of QEMU and not seL4, I would definitely remove this comment, as it is very confusing and doesn't help the user.

Reply:
I can remove this again, but this is really intended for people who try to make more special QEMU configurations work. I have spent some time on this topic, just to see at some point, that the `virt` platform does not allows some things. Background it, that it not meant to be a generic ARM emulator, but implements the ARM variant of the "virt" machine that the QEMU developer came up with. It could be much more generic, but there is some resistance in the QEMU community to allow that.

Reply:
Yes, but leave it to QEMU to give errors on launch. A mostly hidden comment in some CMake file is not going to help anyone. And if it is ever added to QEMU, the comment becomes out of date.

Reply:
removed it